src/LazyOperations/LinearMap.jl

```diff
@@ -108,6 +108,7 @@ struct LinearMap{N<:Real, S<:LazySet{N},
 end
 
 isoperationtype(::Type{<:LinearMap}) = true
+isconvextype(::Type{LinearMap{N, S}}) where {N, S} = isconvextype(S)
```

Comment:
This is wrong: the type argument should either specify *all* the parameters, *or* use `<:` as in `Type{<:LinearMap{N, S}}`.